File: web/src/main/kotlin/no/ssb/kostra/web/viewmodel/ErrorDetailsVm.kt
package no.ssb.kostra.web.viewmodel

import io.micronaut.core.annotation.Introspected

@Introspected
data class ErrorDetailsVm(
    val journalnummer: String,
    val saksbehandler: String,
    val kontrollnummer: String,
    val kontrolltekst: String,
    val feilkode: KostraErrorCode
)

File: web/src/main/vite-project/src/kostratypes/errorDetailsVm.ts
import {KostraErrorCode} from "./kostraErrorCode";

export interface ErrorDetailsVm {
    readonly journalnummer: NonNullable<string>
    readonly saksbehandler: NonNullable<string>
    readonly kontrollnummer: NonNullable<string>
    readonly kontrolltekst: NonNullable<string>
    readonly feilkode: NonNullable<KostraErrorCode>
}